<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifLog</class>
 <widget class="QDialog" name="ModifLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/projetimages/projetimages/e.png</normaloff>:/projetimages/projetimages/e.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">

QLineEdit
{

    border: 1px solid gray;
    border-radius: 4px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
QDialog
{
		background-image: url(:/projetimages/projetimages/a.jpg);

}

*{
font-family:centry gothlic ;
font-size:15px;
}
QWidget#widget{

	background:rgba(0,0,0,0.7);
border-radius:15px;
}


QPushButton
 {
     background-color: red;

     border-radius: 10px;

 }






#label_2{
color:white;
background:transparent;

}

#label{
color:white;
background:transparent;

}





QPushButton
{
color:white;

border-raduis:15px;
}
QPushButton:hover
{

color: white;
border-radius:15px;
background:rgb(255, 85, 127);
}
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>441</width>
     <height>261</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="prixlogmodif_txt">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Type de logement :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="typelogmodfi_txt">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="modifiertypelog_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enregistrer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Prix :</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
